Gruvo/Gruvo/ClientApp/src/app/login-page/login-page.component.html

```diff
@@ -0,0 +1,26 @@
+<link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet">
+<div class="main">
+  <div class="container header">
+    <a class="navbar-brand">Gruvo</a>
+    <gr-login-form class="pull-right"></gr-login-form>
+  </div>
+  <div class="content container">
+
+      <div class="intro container">
+        <p>
+          Share your thoughts all around the world
+        </p>
+        <p>
+          Be the first to find out what is happening in the life of your friends
+        </p>
+
+      </div>
+      <gr-signup-form class="container"></gr-signup-form>
```

Comment:
it's not good idea set some class for component by this way. In this case it's simpler and better input gr-signup-form into div or other wrapper and set class="container" for this wrapper

as I understood, it means that login and registration are located on the same rule?
It seems like we should have login page with **link** to registration which reference to other page where the sign up is